testsuite/MDAnalysisTests/coordinates/test_netcdf.py

```diff
@@ -876,6 +887,133 @@ def test_write_u(self, pos, vel, force, tmpdir, u1, u2):
         u.trajectory.close()
 
 
+class TestNCDFWriterScaleFactors:
+    """Class to check that the NCDF Writer properly handles the
+    application of scale factors"""
+
+    @pytest.fixture()
+    def outfile(self, tmpdir):
+        return str(tmpdir) + 'ncdf-write-scale.ncdf'
```

Comment:
Doesn't
```python
return str(tmpdir / 'beetroots.ncdf')
```
also work?

Reply:
Yeah I can just use f-strings too, I can update (I think I just copy pasted from some fixture in another class somewhat 😅 )

Reply:
`tmpdir` is actually py.path object, not pathlib so I am not even sure if it actually overloads `/`. It might only use `tmpdir.join("cabbage.ncdf")`.

You know what, just leave it. Eventually we get MDA to read path-like objects and then we rewrite it anyway.

Reply:
ok, I'll leave it